### PR TITLE
Added empty homepage information

### PR DESCRIPTION
**Pull request by d2dyno1**
**Details of Changes**
Adds a nice easter egg to the Homepage and hyperlink to settings when no widgets are enabled.

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**
image

### CODE REVIEW

**Comment by d2dyno1** [comment]
Ready!

File: Files/Views/YourHome.xaml.cs [hunk 1]
```diff
@@ -1,35 +1,63 @@
...
+        }
+        
+        public string HomePageEmptyPageSubtitleP1Text
+        {
+            get => "HomePageEmptyPageSubtitleP1".GetLocalized().Replace(' ', (char)160) + "\ufeff";
+        }
+        
+        public string HomePageEmptyPageSubtitleP2Text
+        {
+            get => "HomePageEmptyPageSubtitleP2".GetLocalized().Replace(' ', (char)160) + "\ufeff";
```
Comment (d2dyno1) [inline Files/Views/YourHome.xaml.cs:44]: This had to be done, because xaml textblocks cut off last character(s) if it's space ` `

Comment (d2dyno1) [reply Files/Views/YourHome.xaml.cs:44]: No, `xml:preserve="space"` doesn't work.

Comment (tsvietOK) [reply Files/Views/YourHome.xaml.cs:44]: What is the purpose of space at the end of line?

Comment (d2dyno1) [reply Files/Views/YourHome.xaml.cs:44]: @tsvietOK the ` ` is for the space between text and hyperlink.

HomePageEmptyPageSubtitleP1Text + HyperlinkButton + HomePageEmptyPageSubtitleP2Text

**Comment by yaichenbaum** [comment]
These changes haven't been approved for development, these changes are ok, but they shouldn't be done until users can adjust the widgets from the homepage themselves #4116.

**Comment by yaichenbaum** [comment]
@zeealeid Any ideas to improve the design?

**Comment by zeealeid** [comment]
I see, this screen is supposed to appear when there are no widgets in the home page right?

So maybe we could spice it up a bit by maybe adding an illustration and making the settings button a bit more pronounced, if that makes sense :) 

I'm not on my PC right now so I'll share a quick mock-up soon to explain it better.

**Comment by yaichenbaum** [comment]
> I see, this screen is supposed to appear when there are no widgets in the home page right?
> 
> So maybe we could spice it up a bit by maybe adding an illustration and making the settings button a bit more pronounced, if that makes sense :)
> 
> I'm not on my PC right now so I'll share a quick mock-up soon to explain it better.

Since this PR, the option to choose which widgets are invisible has been move to the toolbar.

**Comment by zeealeid** [comment]
Here's a rough mockup of what I meant :)

Illustration is from https://undraw.co/ (Empty Street) but I slightly edited the colors, maybe use a version of this illustration with brighter colors for light mode?

Frame 503

**Comment by yaichenbaum** [comment]
@zeealeid It will need to be adjusted a little to work with the new flyout for managing widgets, but it looks really nice!

**Comment by R3voA3** [comment]
Could also use a teaching tip control to point users to the widget settings in the toolbar.

image

**Comment by yaichenbaum** [comment]
@d2dyno1 Can we get an update on this PR?

**Comment by d2dyno1** [comment]
@yair100 I've stopped my work on it because I don't have any instructions on it. Moreover I don't have the image that Zee used.

**Comment by yaichenbaum** [comment]
@d2dyno1 The design from @zeealeid is what we should aim for, the only difference is, that instead of opening the settings page, the button should open the widgets flyout.

**Comment by d2dyno1** [comment]
:shipit:

**Comment by d2dyno1** [comment]
I'll close it for now

**Pull request closed without merging**